docs/generate_example_images.py

```diff
@@ -0,0 +1,68 @@
+import importlib
+from pathlib import Path
+
+import cv2
+import numpy as np
+import requests
+import torch
+
+import kornia as K
+
+mod = importlib.import_module("kornia.augmentation")
+
+BASE_IMAGE_URL: str = "https://raw.githubusercontent.com/kornia/data/main/panda.jpg"
+OUTPUT_PATH = Path(__file__).absolute().parent / "source/_static"
```

Comment:
As a hotfix, a simple copy and paste should work:
```python
OUTPUT_PATH = Path(__file__).absolute().parent / "build/html/_static"
os.makedirs(OUTPUT_PATH, exist_ok=True)
```

Reply:
@shijianjian didn't seem to work